config/initializers/version.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+path = Rails.root.join("public/ref")
+version = File.exist?(path) ? File.read(path).strip : nil
```

Comment:
I wonder if here we should run the code in `bin/setup` if the file doesn't exist?

Usually I run `bin/setup` once when setting up the project, but then I'll run `bin/dev` so the file `public/ref` will never be refreshed.